ilagan_andreijireh/index.html

```diff
@@ -183,6 +183,16 @@ <h3>Add a comment here</h3>
         <h3>Comments</h3>
       </div>
 
+      <div class="container">
+        <h1>Country Information</h1>
+        <div class="search-box">
+            <input type="text" id="country-input" placeholder="Enter country name">
```

Comment:
### IDs should be written in snake case. All lower case separated by underscore. 

References: https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g24da8de92d7_0_5

This also applies in lines 190, 192, and 193 in your HTML file.